cirq-core/cirq/transformers/insertion_sort_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+import cirq.transformers
+
+
+def test_insertion_sort():
+    c = cirq.Circuit(
+        cirq.CZ(cirq.q(2), cirq.q(1)),
+        cirq.CZ(cirq.q(2), cirq.q(4)),
+        cirq.CZ(cirq.q(0), cirq.q(1)),
+        cirq.CZ(cirq.q(2), cirq.q(1)),
+    )
+    got = cirq.transformers.insertion_sort_transformer(c)
```

Comment:
Nit - how about renaming `got` --> `c_sorted` for descriptive naming?

Same in cirq-core/cirq/transformers/target_gatesets/cz_gateset_test.py.